code/datums/components/full_auto_fire.dm

```diff
@@ -137,6 +137,8 @@
 		return
 	if(get_dist(source.mob, target) < 2) //Adjacent clicking.
 		return
+	if(QDELETED(target))
+		return
```

Comment:
How can you click on a qdeleted target?

Reply:
the runtime occurs when a target is destroyed just as you begin autofiring on it, the timing is tight but it can happen.

Reply:
Are you entirely sure the target exists and its `gc_destroyed` has a truthy value instead of `params2turf` having returned `null`? Because that would be pretty unusual.

Reply:
params2turf doesn't get triggered at all when reproducing the runtime